Appraisals

```diff
@@ -1,7 +1,13 @@
-appraise 'rails_6_devise_4' do
+appraise 'rails_7_devise_4' do
   # use gemspec constraints
 end
 
+appraise 'rails_6_devise_4' do
+  gem "actionmailer", ">= 3.2.6", "< 7"
+  gem "actionpack", ">= 3.2.6", "< 7"
+  gem "devise", ">= 3.2", "< 6"
```

Comment:
I missed this: in the appraisals that are named, we should constrain the version number to match the name,

```suggestion
  gem "actionmailer", "~> 6.0"
  gem "actionpack", "~> 6.0"
  gem "devise", "~> 4.0"
```